src/main/java/io/snyk/jenkins/workflow/FoundIssuesCause.java

```diff
@@ -0,0 +1,10 @@
+package io.snyk.jenkins.workflow;
+
+import jenkins.model.CauseOfInterruption;
+
+public final class FoundIssuesCause extends CauseOfInterruption {
+  @Override
+  public String getShortDescription() {
+    return "Snyk detected vulnerability issues.";
```

Comment:
Is it only about vulns, or licenses as well? 
If only vulns - maybe better to rephrase to "Snyk has detected security vulnerabilities in your project"
If licenses as well - do we know which one failed?

Reply:
you're right: it's vuln + licenses issues. I like "Snyk has detected security vulnerabilities in your project" :).